checker/check_result.go

```diff
@@ -87,6 +87,10 @@ type CheckResult struct {
 
 // CreateProportionalScore() creates a proportional score.
 func CreateProportionalScore(b, t int) int {
```

Comment:
nit: b and t are unclear var names, can we use long names

Reply:
agreed. will send a follow-up PR